ods_ci/tasks/Tasks/patch_dsc_cr_operatorv2.robot

```diff
@@ -0,0 +1,237 @@
+
+# Switch To Serving Installation Profile
+#     [Documentation]
+#     Switch Installation Profile    profile=serving
+
+# Switch To Training Installation Profile
+#     [Documentation]
+#     Switch Installation Profile    profile=training
+
+# Switch To Workbench Installation Profile
+#     [Documentation]
+#     Switch Installation Profile    profile=workbench
+
+# Switch To None Installation Profile
+#     [Documentation]
+#     Switch Installation Profile    profile=none
+
+# Verify Core Installation Profile
+#     [Documentation]
+#     Run Keyword And Continue On Failure    Verify Component Resources    component=dashboard
+#     Component Should Be Enabled    dashboard
+#     Component Should Not Be Enabled    datasciencepipelines
+#     Component Should Not Be Enabled    distributedWorkloads
+#     Component Should Not Be Enabled    kserve
+#     Component Should Not Be Enabled    modelmeshserving
+#     Component Should Not Be Enabled    workbenches
+
+# Verify Serving Installation Profile
+#     [Documentation]
+#     Run Keyword And Continue On Failure    Verify Component Resources    component=datasciencepipelines
+#     Component Should Be Enabled    datasciencepipelines
+#     Component Should Not Be Enabled    dashboard
+#     Component Should Not Be Enabled    distributedWorkloads
+#     Component Should Not Be Enabled    kserve
+#     Component Should Not Be Enabled    modelmeshserving
+#     Component Should Not Be Enabled    workbenches
+
+# Verify Training Installation Profile
+#     [Documentation]
+#     Run Keyword And Continue On Failure    Verify Component Resources    component=modelmeshserving
+#     Component Should Be Enabled    modelmeshserving
+#     Component Should Not Be Enabled    dashboard
+#     Component Should Not Be Enabled    distributedWorkloads
+#     Component Should Not Be Enabled    kserve
+#     Component Should Not Be Enabled    modelmeshserving
+#     Component Should Not Be Enabled    workbenches
+
+# Verify Workbench Installation Profile
+#     [Documentation]
+#     Run Keyword And Continue On Failure    Verify Component Resources    component=workbenches
+#     Component Should Be Enabled    workbenches
+#     Component Should Not Be Enabled    dashboard
+#     Component Should Not Be Enabled    distributedWorkloads
+#     Component Should Not Be Enabled    kserve
+#     Component Should Not Be Enabled    modelmeshserving
+#     Component Should Not Be Enabled    datasciencepipelines
+
+# Verify None Installation Profile
+#     [Documentation]
+#     Run Keyword And Continue On Failure    Verify Component Resources    component=none
+#     Component Should Not Be Enabled    datasciencepipelines
+#     Component Should Not Be Enabled    dashboard
+#     Component Should Not Be Enabled    distributedWorkloads
+#     Component Should Not Be Enabled    kserve
+#     Component Should Not Be Enabled    modelmeshserving
+#     Component Should Not Be Enabled    workbenches
```

Comment:
these are not valid anymore right?

Reply:
Will be moved to a different test file rather than a task